pkg/controllers/gateway/cluster_eventhandler_test.go

```diff
@@ -13,24 +13,25 @@ import (
 	"k8s.io/client-go/util/workqueue"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
-	gatewayv1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
+	v1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
```

Comment:
Probably better to leave this to avoid potential confusion with our own local apis

Reply:
renamed